packages/icons/src/svg/core/comment.svg

```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
+  <g>
+    <path d="M13.697561,-4.45520775e-05 C14.3064337,-0.00374844656 14.8917413,0.229036047 15.3238873,0.646856976 C15.7560334,1.0646779 15.9993633,1.63306256 16,2.22616152 L16,9.80357547 C16,11.0423127 14.9692683,12.0629524 13.697561,12.0629524 L6.76243902,12.0629524 L3.58439024,15.5687137 C2.98097561,16.2695719 2.4897561,16.098515 2.49317073,15.1838356 L2.49317073,12.0629524 L2.30243902,12.0629524 C1.03073171,12.0629524 0,11.0423127 0,9.80357547 L0,2.22616152 C0.000636672352,1.63306256 0.243966605,1.0646779 0.676112676,0.646856976 C1.10825875,0.229036047 1.69356634,-0.00374844656 2.30243902,-4.45520775e-05 L13.697561,-4.45520775e-05 Z M5,5 C4.44771525,5 4,5.44771525 4,6 C4,6.55228475 4.44771525,7 5,7 C5.55228475,7 6,6.55228475 6,6 C6,5.44771525 5.55228475,5 5,5 Z M8,5 C7.44771525,5 7,5.44771525 7,6 C7,6.55228475 7.44771525,7 8,7 C8.55228475,7 9,6.55228475 9,6 C9,5.44771525 8.55228475,5 8,5 Z M11,5 C10.4477153,5 10,5.44771525 10,6 C10,6.55228475 10.4477153,7 11,7 C11.5522847,7 12,6.55228475 12,6 C12,5.44771525 11.5522847,5 11,5 Z"/>
```

Comment:
```suggestion
    <path class="ti-foreground" d="M13.697561,-4.45520775e-05 C14.3064337,-0.00374844656 14.8917413,0.229036047 15.3238873,0.646856976 C15.7560334,1.0646779 15.9993633,1.63306256 16,2.22616152 L16,9.80357547 C16,11.0423127 14.9692683,12.0629524 13.697561,12.0629524 L6.76243902,12.0629524 L3.58439024,15.5687137 C2.98097561,16.2695719 2.4897561,16.098515 2.49317073,15.1838356 L2.49317073,12.0629524 L2.30243902,12.0629524 C1.03073171,12.0629524 0,11.0423127 0,9.80357547 L0,2.22616152 C0.000636672352,1.63306256 0.243966605,1.0646779 0.676112676,0.646856976 C1.10825875,0.229036047 1.69356634,-0.00374844656 2.30243902,-4.45520775e-05 L13.697561,-4.45520775e-05 Z M5,5 C4.44771525,5 4,5.44771525 4,6 C4,6.55228475 4.44771525,7 5,7 C5.55228475,7 6,6.55228475 6,6 C6,5.44771525 5.55228475,5 5,5 Z M8,5 C7.44771525,5 7,5.44771525 7,6 C7,6.55228475 7.44771525,7 8,7 C8.55228475,7 9,6.55228475 9,6 C9,5.44771525 8.55228475,5 8,5 Z M11,5 C10.4477153,5 10,5.44771525 10,6 C10,6.55228475 10.4477153,7 11,7 C11.5522847,7 12,6.55228475 12,6 C12,5.44771525 11.5522847,5 11,5 Z"/>
```

Reply:
@frassinier Can we close the PR or this change is requested?

Reply:
Yes I will do the change

Reply:
Cool, thx 👍